docs/en/development/contrib.md

```diff
@@ -18,7 +18,7 @@ SELECT library_name, license_type, license_path FROM system.licenses ORDER BY li
 Note that the listed libraries are the ones located in the `contrib/` directory of the ClickHouse repository.
 Depending on the build options, some of the libraries may have not been compiled, and, as a result, their functionality may not be available at runtime.
 
-[Example](https://play.clickhouse.com/play?user=play#U0VMRUNUIGxpYnJhcnlfbmFtZSwgbGljZW5zZV90eXBlLCBsaWNlbnNlX3BhdGggRlJPTSBzeXN0ZW0ubGljZW5zZXMgT1JERVIgQlkgbGlicmFyeV9uYW1lIENPTExBVEUgJ2VuJw==)
+[Example](https://sql.clickhouse.com/play?query_id=478GCPU7LRTSZJBNY3EJT3)
```

Comment:
Query is missing for some reason

<img width="1042" alt="image" src="https://github.com/user-attachments/assets/e71c7159-cc8a-42e0-a40f-032012a96b4a">

Reply:
@vdimir this is pending a DNS deployment of a new UI, hence draft